lib/actions/local/configure.coffee

```diff
@@ -95,8 +66,48 @@ inquirerOptions = (data) -> [
 	}
 ]
 
+# Since these paths are used by `resin-image-fs`, we can safely
+# assume they are always UNIX style.
+joinPath = (path) -> '/' + path.join('/')
```

Comment:
Manually joining strings look fragile. Node.js provides `path.join()` in the `path` built-in module (`require('path')`). If we always want to use UNIX-style, then you can use `path.posix.join()`.

Reply:
Maybe something like this?

```js
path = require('path')
joinPath = (fragments) -> path.posix.join(path.posix.sep, ...fragments)
```